### PR TITLE
Filter, conversion, metadata documentation

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: docs/Quick_start_guide.rst [hunk 1]
```diff
@@ -65,46 +90,147 @@ Types can also be obtained from their name in the C programming language:
...
+  NDArrayITKBase([[    5,  -997],
+                  [    5, -1003],
+                  [ -993,  -999],
+                  [ -996,  -994]], dtype=int16)
+
+Input/Output (IO)
+.................
+
+Convenient functions are provided read and write from ITK's many supported
+file formats.::
```
Comment (jhlegarreta) [inline docs/Quick_start_guide.rst:149]: The period is probably not necessary.

File: docs/Quick_start_guide.rst [hunk 2]
```diff
@@ -65,46 +90,147 @@ Types can also be obtained from their name in the C programming language:
...
-Similar functions are available to work with `itk.Matrix`, VNL vectors and matrices:
+.. literalinclude:: code/ExplicitInstantiation.py
+   :lines: 8-
 
-.. literalinclude:: code/MixingITKAndNumPy.py
-   :lines: 35-
+Instantiate an ITK object
+.........................
+
+There are two types of ITK objects. Most ITK objects (images, filters, adapters, ...) are instantiated the following way:
```
Comment (jhlegarreta) [inline docs/Quick_start_guide.rst:223]: I'd avoid using the ellipsis. If there is not an easy way to list all objects, I'd put it as "Most ITK objects (such as images, filters, or adapters, among others) are  (...)", or "Most ITK objects (such as images, filters, or adapters, for example) are  (...)" or simply "Most ITK objects (such as images, filters, or adapters) are  (...)"

File: docs/Quick_start_guide.rst [hunk 3]
```diff
@@ -65,46 +90,147 @@ Types can also be obtained from their name in the C programming language:
...
+
+.. literalinclude:: code/InstantiateITKObjects.py
+   :lines: 6-8
 
+Some objects (matrix, vector, RGBPixel, ...) do not require the attribute `.New()` to be added to instantiate them:
```
Comment (jhlegarreta) [inline docs/Quick_start_guide.rst:228]: Same comment as above.

**Review by jhlegarreta — COMMENTED** [review]
Thanks for doing this Matt.

File: docs/Quick_start_guide.rst (hunk 1 above)
Comment (thewtex) [reply docs/Quick_start_guide.rst:149]: :+1: removed

File: docs/Quick_start_guide.rst (hunk 2 above)
Comment (thewtex) [reply docs/Quick_start_guide.rst:223]: :+1:

File: docs/Quick_start_guide.rst (hunk 3 above)
Comment (thewtex) [reply docs/Quick_start_guide.rst:228]: Agreed, modified.

**Review by jhlegarreta — APPROVED** [review]
:100:

**Comment by thewtex** [comment]
Thanks for the review @jhlegarreta !

**Pull request merged**